<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBackupTargetDialog</class>
 <widget class="QDialog" name="AddBackupTargetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Backup Target</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Target Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ValidatingLineEdit" name="target_name_field"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Target Path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ValidatingLineEdit" name="target_path_field"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="path_picker_button">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="folder-open">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Default Backup Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ValidatingLineEdit" name="default_backup_field"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>First Backup Name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="first_backup_field"/>
   </item>
   <item row="4" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>345</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ui/validatinglineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddBackupTargetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddBackupTargetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
